### PR TITLE
MC Updates

### PR DESCRIPTION
**Pull request by edwardloveall**
Just some tips to help new MCs compiled from our pre-workshop scrambling

### CODE REVIEW

File: mc/mc-tips.md [hunk 1]
```diff
@@ -34,6 +34,27 @@
...
+
+* Welcome everyone to RailsBridge
+  * Welcome is probably along the lines of encouraging people to get food if they haven’t yet, find a comfortable spot, log into Wi-Fi, etc. At this point people may still be arriving based on when they get out of work/how far they have to travel
+* Present a "when you're done installing" slide to set expectations for the night
+  * This might be helpful in terms of framing what the goals of the night and the weekend are. That the goal of Fri is basically "get your stuff up and running so that we can start on the rails curriculum tomorrow", and "learn a little bit about the command line". Sometimes people are confused because they don’t actually start learning ruby until Saturday
+* Thank you to the sponsors
+  * Awesome, sponsors! There might be some overlap between TAs and people representing sponsors. In which case probably the sponsor intro is more about what the company does/what their stack is/are they hiring/whatever.
+* Invite sponsors to the stage to talk about their company
+  * Invite TAs up. Your name, pronouns, what you love about being in tech, and who is someone who helped you on your coding journey?
```
Comment (SViccari) [inline mc/mc-tips.md]: Is this intentional? To have "Invite TAs up" nested within "Invite sponsors to the stage...."? Perhaps we can remove this line as it seems unrelated.

File: mc/mc-tips.md [hunk 2]
```diff
@@ -34,6 +34,27 @@
...
+* Thank you to the TAs
+* Invite the TAs to the stage for individual introductions
+  * Your name, pronouns, what you love about being in tech, and who is someone who helped you on your coding journey?
```
Comment (SViccari) [inline mc/mc-tips.md]: What do you think of limiting the TA intros to just one personal question? Alternative:
> Example Intro: Name, pronouns, and one question. Possible questions are: what do you love about being in tech?  Who is someone who helped you on your coding journey?

File: mc/mc-tips.md [hunk 3]
```diff
@@ -34,6 +34,27 @@
...
+  * Lead into what the role of TAs are. They are here for you to ask questions, help you get unstuck, debug with you. And that starts with installfest!
+* Introduce Installfest and and have a slide showing where to go (`docs.railsbridgeboston.org`)
+* Introduce the "Intro the command line" talk
+  * Poll TAs when we get about 50% of people up and running. At this point we can pause for the command line talk. The TA coordinator will know who's giving it. Then the people who are done can peace out afterwards, and those who still have installing to do can continue with help from TAs.
+* Keep the workshop going and let people know they can leave when they're done
+  * After the Intro to CL talk might be a good time for you to go up, thank the presenter, and show the "when you’re done installing" slide, because at that point people can start leaving if they’re done or sticking around if they need more help installing.
+* Close out the night and remind everyone that breakfast starts at 9:30am
```
Comment (SViccari) [inline mc/mc-tips.md]: Possible alternative to avoid updating these docs should the breakfast time change:
> Close out the night and remind everyone what time breakfast will be delivered and when the workshop begins

File: mc/mc-tips.md [hunk 4]
```diff
@@ -34,6 +34,27 @@
...
+* Thank you to the TAs
+* Invite the TAs to the stage for individual introductions
+  * Your name, pronouns, what you love about being in tech, and who is someone who helped you on your coding journey?
+  * Lead into what the role of TAs are. They are here for you to ask questions, help you get unstuck, debug with you. And that starts with installfest!
+* Introduce Installfest and and have a slide showing where to go (`docs.railsbridgeboston.org`)
+* Introduce the "Intro the command line" talk
+  * Poll TAs when we get about 50% of people up and running. At this point we can pause for the command line talk. The TA coordinator will know who's giving it. Then the people who are done can peace out afterwards, and those who still have installing to do can continue with help from TAs.
+* Keep the workshop going and let people know they can leave when they're done
+  * After the Intro to CL talk might be a good time for you to go up, thank the presenter, and show the "when you’re done installing" slide, because at that point people can start leaving if they’re done or sticking around if they need more help installing.
```
Comment (SViccari) [inline mc/mc-tips.md]: Possible alternative:
> * Introduce the "Intro the Command Line" talk
>   * Poll TAs.... etc...
> * Thank the presenter and show a link to the presenter's slides (if available). 
> * Keep the workshop going and let people know they can leave when they're done
>   * Show a "when you’re done installing" slide to encourage attendees to continue working through the install instructions or, if they're done, they're can head home.

**Comment by SViccari** [comment]
@edwardloveall Thank you for capturing this helpful information!

File: mc/mc-tips.md [hunk 5]
```diff
@@ -34,6 +34,27 @@
 * Start the workshop!
 * Keep the slide with the internet password on the board.
 
+### Typical Installfest Announcements
+
+* Welcome everyone to RailsBridge
+  * Welcome is probably along the lines of encouraging people to get food if they haven’t yet, find a comfortable spot, log into Wi-Fi, etc. At this point people may still be arriving based on when they get out of work/how far they have to travel
+* Present a "when you're done installing" slide to set expectations for the night
+  * This might be helpful in terms of framing what the goals of the night and the weekend are. That the goal of Fri is basically "get your stuff up and running so that we can start on the rails curriculum tomorrow", and "learn a little bit about the command line". Sometimes people are confused because they don’t actually start learning ruby until Saturday
```
Comment (mxie) [inline mc/mc-tips.md]: 👍 setting expectations early on

File: mc/mc-tips.md [hunk 6]
```diff
@@ -34,6 +34,27 @@
 * Start the workshop!
 * Keep the slide with the internet password on the board.
 
+### Typical Installfest Announcements
+
+* Welcome everyone to RailsBridge
+  * Welcome is probably along the lines of encouraging people to get food if they haven’t yet, find a comfortable spot, log into Wi-Fi, etc. At this point people may still be arriving based on when they get out of work/how far they have to travel
```
Comment (mxie) [inline mc/mc-tips.md]: For a lot of these tips, I wonder if this would be easier to follow if we broke some of these lists into bulleted lists instead. For example:

```suggestion
  * Encouraging people to:
    - get food if they haven’t yet
    - find a comfortable spot
    - log into Wi-Fi, etc.
  * At this point people may still be arriving based on when they get out of work/how far they have to travel
```

I don't know - something like that. I think I just personally don't like fully-written paragraphs 😅 

WDYT?

File: mc/mc-tips.md [hunk 7]
```diff
@@ -34,6 +34,27 @@
...
+* Present a "when you're done installing" slide to set expectations for the night
+  * This might be helpful in terms of framing what the goals of the night and the weekend are. That the goal of Fri is basically "get your stuff up and running so that we can start on the rails curriculum tomorrow", and "learn a little bit about the command line". Sometimes people are confused because they don’t actually start learning ruby until Saturday
+* Thank you to the sponsors
+  * Awesome, sponsors! There might be some overlap between TAs and people representing sponsors. In which case probably the sponsor intro is more about what the company does/what their stack is/are they hiring/whatever.
```
Comment (mxie) [inline mc/mc-tips.md]: Sometimes it's also nice to have them include why they were interested in being involved. Completely optional though - I'm totally ok with people just being generous with their 💵 too ;)

File: mc/mc-tips.md [hunk 8]
```diff
@@ -34,6 +34,27 @@
...
+* Invite sponsors to the stage to talk about their company
```
Comment (mxie) [inline mc/mc-tips.md]: Is this meant to be separate from the previous bullet points?

File: mc/mc-tips.md (hunk 2 above)
Comment (mxie) [reply mc/mc-tips.md]: I like it!

File: mc/mc-tips.md [hunk 9]
```diff
@@ -34,6 +34,27 @@
...
+  * Invite TAs up. Your name, pronouns, what you love about being in tech, and who is someone who helped you on your coding journey?
+* Thank you to the TAs
+* Invite the TAs to the stage for individual introductions
+  * Your name, pronouns, what you love about being in tech, and who is someone who helped you on your coding journey?
+  * Lead into what the role of TAs are. They are here for you to ask questions, help you get unstuck, debug with you. And that starts with installfest!
```
Comment (mxie) [inline mc/mc-tips.md]: Would it make sense to have this explanation happen before the TAs introduce themselves? This might help students understand why they're listening to people introduce themselves and tell their stories.

File: mc/mc-tips.md (hunk 4 above)
Comment (mxie) [reply mc/mc-tips.md]: I dig it.

File: mc/mc-tips.md (hunk 6 above)
Comment (edwardloveall) [reply mc/mc-tips.md]: Fair enough! I don't think I have a strong opinion either way. These were mostly copy/paste from Slack and I didn't change the format. I'll do my best to bullet them.

File: mc/mc-tips.md (hunk 8 above)
Comment (edwardloveall) [reply mc/mc-tips.md]: Nah, I just messed it up.

File: mc/mc-tips.md (hunk 9 above)
Comment (edwardloveall) [reply mc/mc-tips.md]: I think that's covered more-or-less in the second step

>Present a "when you're done installing" slide to set expectations for the night

I'll make that more explicit though.

Comment (edwardloveall) [reply mc/mc-tips.md]: Actually yeah it's all there in the sub-bullet

File: mc/mc-tips.md [hunk 10]
```diff
@@ -34,6 +34,54 @@
...
+
+Poll TAs: when we get about 50% of people up and running. At this point we can pause for the command line talk. The TA coordinator will know who's giving it. Then the people who are done can peace out afterwards, and those who still have installing to do can continue with help from TAs.
+
+* Thank the presenter and show a link to the presenter's slides (if available).
+* Keep the workshop going and let people know they can leave when they're done
+  * Show a "when you’re done installing" slide
+    * Encourage attendees to continue working through the install instructions
+    * If they're done, they're can head home
+* Close out the night and remind everyone
+  * what time breakfast will be delivered
```
Comment (SherrySeibel) [inline mc/mc-tips.md]: "what time breakfast will be served" might be better. Breakfast will be delivered much earlier than when the students arrive.
